### DISCUSSION TITLE
"MinRisk" and "MaxRet"

### DISCUSSION THREAD

**Discussion by REICHIYAN** [dcajasn/Riskfolio-Lib#91] [Q&A] [answered]
Dear Mr. @dcajasn 

Excuse my rudimentary question, but I have a question about the [Optimization](https://riskfolio-lib.readthedocs.io/en/latest/portfolio.html#Portfolio.Portfolio.optimization) method.

We can select 4 different obj. 

I hane a question  about the formulation when "MinRisk" and "MaxRet" are selected.

I referred to the paper "Entropic Portfolio Optimization: a Disciplined Convex Programming Framework".
[delivery.php.pdf](https://github.com/dcajasn/Riskfolio-Lib/files/9079002/delivery.php.pdf)

I believe "MinRisk" is equation (16) and "MaxRet" is equation (15). Is my understanding correct?

Best regards,
Rei

**Comment by dcajasn** [comment, marked as answer]
Hi @REICHIYAN 

Yes, you are correct.

Best,
Dany

**Reply by REICHIYAN** [reply]
Hi Mr. @dcajasn

Thank you for your prompt response!

Best regards,
Rei
